main.go

```diff
@@ -28,6 +29,15 @@ func main() {
 		log.Panic(err)
 	}
 
+	sdaDB, err := common.NewSDAdb(config.DB)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
```suggestion
		log.Panic(err)
```
Fatal prevents any downstream defer from being excuted